docs/How_to_Guides/authentication.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Authentication
+sidebar_position: 1
+---
+
+# Authentication
+
+In order to push or pull artifacts or images to an OCI registry, we need to prove our identity using authentication. Following this, authorization provides the required permissions to the user.
+
+In ORAS, we can use [`oras login`](../CLI_Reference/oras_login.mdx) in advance to authenticate ourselves for private registries. By default, this command stores the credentials in `~/.docker/config.json` (same file used by the docker client). If you have previously authenticated to a registry using `docker login`, the credentials will be reused.
+
+You can use the `-c/–config` option to specify an alternate location.
+
+:::info
+
+While ORAS leverages the local docker client config store, ORAS does NOT have a dependency on Docker Desktop running or being installed. ORAS can be used independently of a local docker daemon.
+
+:::
+
+`oras` also accepts explicit credentials via options, for example,
+
+```
+oras pull -u username -p password myregistry.io/myimage:latest
+```
+
```

Comment:
Could have a blurb here about how the disadvantage of this approach is you need to authenticate for each command rather than once for multiple commands.